### DISCUSSION TITLE
Smoke detector that work in series even if Z2M controller/HA server fails?

### DISCUSSION THREAD

**Discussion by Naguz** [Koenkk/zigbee2mqtt#15245] [Devices]
Does anyone know of a smoke alarm that can be connected in series and work also if the controller fails? The Develco SMSZB-120 claims to do this, but it does not. Having smoke detectors where you are not guaranteed that everyone goes off if one goes off is a downgrade from regular "dumb" smoke detectors connected in series, and not very smart. This could be fixed with an automation, but I'm not very comfortable with needing both Z2M and HA to be running to fire off an automation alerting the other smoke detectors if one of them detects smoke. What if the fire started in my HA server? It is very unlikely, but fire safety should work 100% of the time, and be as little complicated as possible. 

To me this could/should work as easily as the smoke detectors having a group binding to the same group, but group binding does not seem to be a feature available smoke detectors are quipped with. 

It would also be a big plus if the alarm can be triggered manually via Z2M, for other uses (burglary alarm t.ex.)

**Comment by Miwer** [comment]
Hi @Naguz - I realize it's been a couple of years since you created this thread. Did you ever manage to find a product that fits the purpose?

I too am looking for a smoke detector that can bind with other detectors via zigbee groups, and sound the alarm on all detectors, should one be triggered (in case the coordinator is offline).

It seems like it could be a selling point for smoke detectors, but I can find no device that advertise support for this. Maybe it falls into the advanced category of zigbee use, that is not easily configured out of the box.

**Reply by Naguz** [reply]
What I found is that several of the zigbee enabled smoke alarms does this, even if they claim to need a hub to do it. This device has been successfully tested by another usr on a forum I'm on with zigbee routered powered off: https://proff.elko.no/elko-smart/ELKO%20-%20Smart%20Smoke%20Alarm%20Battery_Device%20user%20guide/English/ELKO%20-%20Smart%20Smoke%20Alarm%20Battery_Device%20user%20guide.pdf All alarms went off. 

I have also seen people claim that the  SMSZB-120 actually does this if you test it with smoke. As i decided to go with a cheap alarm company for added security both concerning fire and intrusion, I stopped looking for it. Perhaps the best option is to buy a couple from somewhere with a good return policy, connect them, let them sit for a while to do any coordination, and then test, with either a test spray or the test button.

It is frustrating that "requires X hub to work" is such a common disclaimer.

**Comment by slaudel** [comment]
Hi,

has there been any advancement?
I have some [Develco SMSZB-120](https://www.zigbee2mqtt.io/devices/SMSZB-120.html#develco-smszb-120) zigbee enabled smoke detectors which are located around the house on different floors. They are all paired to z2m. Now I'm asking myself, if it would be possible to bind them together? I think it would be possible to do this via my home automation system (openhab) but I would prefer the binding on the lowest level possible, directly on the zigbee level.
What I want is, if one detector is triggered to give smoke alarm, it should trigger all other (or at least one on each floor) smoke detectors to sound the alarm. This should be without the intervention of any other system (z2m, openhab) to make it as reliable as possible.
Would this be possible with the smoke detector and if so, how would I do this? I've seen the binding settings for the devices, but I don't know what endpoints needs to be bound to what endpoint on the remote devices/groups and if this will work at all?

**Reply by Miwer** [reply]
> Hi,
> 
> has there been any advancement? I have some [Develco SMSZB-120](https://www.zigbee2mqtt.io/devices/SMSZB-120.html#develco-smszb-120) zigbee enabled smoke detectors...

Hi @slaudel. Well, if you already have them, feel free to experiment with the binding settings, see if you can bind some of them together, and have them trigger each other. You could try and create a group in z2m and then bind a couple of smoke detectors to that group. As to what endpoints to use, well, you'll have to experiment there. Whatever your findings are, we would very much appreciate if you share them with us, even if you do not get it working. :smile:
